WebFileSystem.ts

```diff
@@ -586,13 +586,13 @@ class WebFileSystem extends webdav.FileSystem {
                             callback(error)
                         }
                     } else {
-                        logger.error('Directory could not be found!')
+                        logger.error(`WebFileSystem._readDir.loadPath.false : Directory could not be found! uid: ${user.uid} path: ${path.toString()}`, new Error('Stack-Tracer'))
                         callback(webdav.Errors.ResourceNotFound)
                     }
                 }
             }
         } else {
-            logger.error(webdav.Errors.BadAuthentication.message)
+            logger.error(`WebFileSystem._readDir.context.user.false : ${webdav.Errors.BadAuthentication.message} path: ${path.toString()}`, new Error('Stack-Tracer'))
```

Comment:
The other `Bad Authentication`-outputs were warnings, this one is an error, should be consistent